test/test_manifest.json

```diff
@@ -3213,6 +3213,12 @@
       "rounds": 1,
       "type": "eq"
     },
+    {  "id": "js-authors",
+       "file": "pdfs/js-authors.pdf",
+       "md5": "e892f290ae209286a29d3c17dfeab226",
+       "rounds": 1,
+       "type": "eq"
+    },
```

Comment:
How much value does this `eq` test bring, since the functionality that you're adding here doesn't really show up in a reference-test as far as I can tell?